src/main/java/org/springframework/data/elasticsearch/client/elc/RequestConverter.java

```diff
@@ -968,6 +968,72 @@ public DeleteByQueryRequest documentDeleteByQueryRequest(Query query, @Nullable
 		});
 	}
 
+	public DeleteByQueryRequest documentDeleteByQueryRequest(DeleteQuery query, @Nullable String routing, Class<?> clazz,
+															 IndexCoordinates index, @Nullable RefreshPolicy refreshPolicy) {
+		Assert.notNull(query, "query must not be null");
+		Assert.notNull(index, "index must not be null");
+
+		// Old documentDeleteByQueryRequest needs to be refactored.
```

Comment:
is that comment still relevant?

Reply:
In my opinion, it would be better to mark it as _TODO_ because I want to avoid refactoring due to the update and delete Query, as well as the previous delete by query.